specs/core/1_custody-game.md

```diff
@@ -126,7 +126,7 @@ This document details the beacon chain additions and changes in Phase 1 of Ether
     'attestation': Attestation,
     'challenger_index': ValidatorIndex,
     'responder_key': BLSSignature,
-    'chunk_bits': Bitfield,
+    'chunk_bits': "bytes",
```

Comment:
```suggestion
    'chunk_bits': 'bytes',
```
Side question: Do we want a bitfield custom type?

Reply:
This will be implemented in #1096

Reply:
Yup! I would like to keep this PR this way for now though as otherwise the tests fail (and I am rapidly reaching my failing-test exposure saturation point).